__tests__/app.test.js

```diff
@@ -21,3 +21,29 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: should return an array of topics objects when topics exist", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics).toHaveLength(data.topicData.length);
```

Comment:
The importing in "data" here is a good idea. I would probably only do this if I _knew_ that I was going to add more data to the original, otherwise I would just put a number in here. Happy either way.